website/src/site/site.xml

```diff
@@ -126,7 +125,8 @@ under the License.
 		        of The Apache Software Foundation. All other marks mentioned may be trademarks or registered
 		        trademarks of their respective owners.
 	        </div>
-        </footer>                
+            <script src="js/release.js" type="text/javascript"></script>
```

Comment:
This way (and with added call to `setReleaseLink` in the end of release.js) information about the latest version is updated using js after it's loaded. I think this could be useful if for some reason the information in downloads.md is incorrect. But we keep information there also in case people without JavaScript use website